user_manual/android_app.rst

```diff
@@ -2,202 +2,305 @@
 
-.. figure:: images/android-4.png 
-   :alt: Your ownCloud Files page.
-   
-Click the overflow button at the top right (that's the one with three vertical 
-dots) to open a user menu. **Grid view** toggles between grid and list 
-view. **Refresh account** syncs with the server, and **Sort** 
-gives you the option to sort your files by date, or alphabetically.
+- Manage Users Accounts
+- Current **Uploads** 
+- All Files
+- Application **Settings**
+
+Manage Users Accounts
+~~~~~~~~~~~~~~~~~~~~~
+
+Initially the path to this section isn't visible. 
+To get to it, first click the down arrow, in the user details section, which
+will replace the "All files" and "Uploads" buttons with "Add account" and
+"Manage accounts". 
+Then, click "Manage accounts". 
+From there, you can see all of the currently active user accounts, along with a button to add a new account.
+
+User Accounts
+^^^^^^^^^^^^^
+
+After clicking "Manage Accounts", you will see a list of the currently active
+accounts in the application, as in the screenshot below. 
+Each entry in the list has shortcuts to:
+
+- Viewing the user's files
+- Changing the user's password
+- Removing the account
 
 .. figure:: images/android-6.png
    :alt: Top-right menu.   
 
-The ownCloud menu at the top left shows the name of the logged-in user, has a 
-shortcut to your files, the **Uploads** button, and the **Settings** button.
+Change User Password
+^^^^^^^^^^^^^^^^^^^^
+
+To change a user's password, click the key icon, next to the user's details. 
+This will display the user details page, with the ownCloud server URI and user account pre-filled.
+Enter a new password, and click "Connect", and the password will be updated.
+
+.. NOTE:: 
+   If you want extra security, you can set a login PIN on your Android device,
+   and also on your ownCloud account. If you are using a shared Android device,
+   other users can access your files in the file manager if you are sharing
+   a single user account. To avoid this, you could set up multiple user
+   accounts to protect your files.
+
+.. figure:: images/android-13.png
+   :alt: Change password or remove account dialog.
+
+Removing An Account / Logging Out
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Uploads page displays the status of files currently uploading, a list of 
-your recently-uploaded files, and a Retry option for any failed uploads. If 
-credentials to access the file have changed, you'll see a credentials error. 
-Tap the file to retry, and you'll get a login screen to enter the new 
-credentials. 
+To remove an account, click the rubbish bin icon, next to the key icon. 
+This will display a confirmation dialog, asking if you want to remove the account.
+If you click "Yes", the account will be removed. 
+This action logs both you out of the server and removes your account settings **and**
+files from your Android device. 
```

Comment:
Thanks for the clarification. I'm not sure if I took this information from somewhere else, and modified it, or assumed it based on my experimentation with the Android client. Either way, I'll updated it.